### PR TITLE
Add new JavaScript questions

### PR DESCRIPTION
**Pull request by amehi0index**
Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,48 @@
 const fullQuiz = [
+  {
+    Question:
+      "A variable that has been declared, but has not been assigned a value is _______________",
+    Answer: "undefined",
+    Distractor1: "unknown",
+    Distractor2: "unassigned",
+    Distractor3: "pending",
+    Explanation:
+    "A variable that has not been assigned a value is of type undefined",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:10]: Proposed fix:
In JavaScript, a variable that has been declared but not assigned a value is known as undefined.

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+  },
+  {
+    Question:
+      "A variable that has been declared and assigned a value of 'no value' is ___________________",
+    Answer: "null",
+    Distractor1: "non-value",
+    Distractor2: "empty",
+    Distractor3: "unreturned",
+    Explanation:
+    "Null value is assigned to a variable to represent that the variable exists, but has no type or value.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:21]: Proposed fix:
In JavaScript, null represents the absence of a value.

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,4 +1,48 @@
...
+  {
+    Question:
+      "In a switch statement, the __________________ case executes if none of the options are true.",
+    Answer: "default",
+    Distractor1: "override",
+    Distractor2: "else",
+    Distractor3: "alert",
+    Explanation:
+    "A default clause is executed if the value of expression doesn't match any of the case clauses.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:44]: Whenever possible we like to use freeCodeCamp resources first before moving to MDN.

Proposed fix:
Link:"https://www.freecodecamp.org/news/javascript-switch-case-js-switch-statement-example/",

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
HI @amehi0index ! 

Thank you for your questions! 
Please review my requested changes 👍

**Comment by jdwilkin4** [comment]
Also, please make sure to update the format for your questions so the key value pairs are in the same line.

Please follow this example:
```
  {
    Question: "What is a maximum length for a filename under Linux?",
    Answer: "255 Bytes",
    Distractor1: "225 Bytes",
    Distractor2: "235 Bytes",
    Distractor3: "295 Bytes",
    Explanation: "The maximum length for a file name is 255 bytes. The maximum combined length of both the file name and path name is 4096 bytes.",
    Link: "https://www.ibm.com/docs/en/spectrum-protect/8.1.9?topic=parameters-file-specification-syntax",
  },
```

**Review by amehi0index — COMMENTED** [review]
Made changes as per your request.

**Comment by jdwilkin4** [comment]
Hi @amehi0index ! 

I am not seeing your recent changes committed in this PR. 
You might need to double check and make sure your changes you made locally are showing up in your remote repo on GitHub. 
Let me know if you have any questions! 😃

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by amehi0index** [comment]
Thank you!

**Pull request merged**